### PR TITLE
Allow to specify view parameter for codesandbox

### PR DESCRIPTION
**Pull request by erykpiast**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Added support for a `view` option to CodeSandbox liquid tag.

## Related Tickets & Documents

--

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

--

## Added tests?

- [x] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## What gif best describes this PR or how it makes you feel?

dance

### CODE REVIEW

File: app/liquid_tags/codesandbox_tag.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@ def parse_options(input)
   # composed of letters, numbers, dashes, underscores, forward slashes, @ signs, periods/dots,
   # and % symbols.  Invalid options will raise an exception
   def valid_option(option)
-    raise StandardError, "CodeSandbox Error: Invalid options" unless (option =~ /\A(initialpath=([a-zA-Z0-9\-\_\/\.\@\%])+)\Z|\A(module=([a-zA-Z0-9\-\_\/\.\@\%])+)\Z|\A(runonclick=((0|1){1}))\Z/)&.zero?
+    raise StandardError, "CodeSandbox Error: Invalid options" unless (option =~ /\A(initialpath=([a-zA-Z0-9\-\_\/\.\@\%])+)\Z|\A(module=([a-zA-Z0-9\-\_\/\.\@\%])+)\Z|\A(runonclick=((0|1){1}))\Z|\Aview=(editor|split|preview)\Z/)&.zero?
```
Comment (rhymes) [inline app/liquid_tags/codesandbox_tag.rb:51]: could we move this giant regexp to a constant up in the class?

Something like:

```ruby
OPTIONS_REGEXP = /stuff/.freeze
```

Comment (erykpiast) [reply app/liquid_tags/codesandbox_tag.rb:51]: It's my first Ruby code I've ever written, but I can definetely do that! :)

Comment (rhymes) [reply app/liquid_tags/codesandbox_tag.rb:51]: wow @erykpiast, thanks a ton then! Don't worry if you find it tricky, the PR is working already (that's why I accepted it).

Welcome to DEV and welcome to Ruby ;-)

**Comment by erykpiast** [comment]
Okay, so I wonder if it's worth changing the code if it has to approvals and was tested 🤔 If it is in your opinion, I'll do that at the evening!

**Comment by rhymes** [comment]
@erykpiast don't worry, thanks again! You can submit another PR if you want to :-)

**Comment by erykpiast** [comment]
Will do!

**Pull request merged**